app/lib/actions/pulp/repository/update_importer.rb

```diff
@@ -1,7 +1,7 @@
 module Actions
   module Pulp
     module Repository
-      class UpdateImporter < Pulp::Abstract
+      class UpdateImporter < Pulp::AbstractAsyncTask
```

Comment:
Is there a reason to change to `AbstractAsyncTask`? I'm trying to refresh my memory on the difference, but I believe an `AbstractAsyncTask` is a pulp task that spawns child tasks.

Reply:
`AbstractAsyncTask` was needed here so it polls and waits on the Pulp task. Without it, it will just fire/forget.